src/components/FolderContent.vue

```diff
@@ -13,7 +13,7 @@
 				/>
 				<NewMessageDetail v-if="newMessage" />
 				<Message v-else-if="showMessage" />
-				<NoMessageSelected v-else-if="hasMessages" />
+				<NoMessageSelected v-else-if="hasMessages && !this.isMobile" />
```

Comment:
```suggestion
				<NoMessageSelected v-else-if="hasMessages && !isMobile" />
```

No need for `this` in a few template. It's added automagically ;)

Reply:
whoops, of course; you're right...